lib/src/pagination/pagination.component.html

```diff
@@ -0,0 +1,101 @@
+<div class="pagination-wrapper" [ngClass]="className" [id]="id || ''">
```

Comment:
if id doesnt exist then dont give it an id, think about what will happen if another element have id=""
i suggest
```typescript
<div class="pagination-wrapper" [ngClass]="className" [attr.id]="id">
 ```
this syntax the id property will not be rendered if there is no id

Reply:
I think if he initializes `id` with an empty string then it will be rendered, but if he leaves it as `undefined` or initializes it with `null` then it's not gonna be rendered

Reply:
It looks like a cleaner solution.